functions/function/twitter-function/src/test/java/org/springframework/cloud/fn/twitter/users/TwitterUsersFunctionTests.java

```diff
@@ -165,7 +164,7 @@ public void testOne() throws IOException {
 
 	@TestPropertySource(properties = {
 			"twitter.users.type=lookup",
-			"twitter.users.lookup.screenName=#jsonPath(payload,'$..[*].code')"
+			"twitter.users.lookup.screenName=#jsonPath(payload,'$[*].code')"
```

Comment:
Interesting enough, the latest libs (I think its JsonPath internally doing this) does not return results when the "deep scan - '..'" is added to the query. Works fine w/o it.

Reply:
I'm not sure that syntax is correct. That's why it fails.
I believe it has to be like this: `$..code` - meaning "all codes".
See examples and their explanation here: https://github.com/json-path/JsonPath#path-examples

Reply:
Hmm. It was working before so maybe they fixed a bug that was allowing that to work.

Reply:
That's my thought, too. 
Even if it worked before it doesn't mean that `..[*]` syntax is correct.